jest.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
```

Comment:
Similar to what we saw in the ionic-native version. We could also implement TypeScript for just the specs to get type helping but I'll only add it if that's the direction you want to take this.